src/haiku.rs

```diff
@@ -2,11 +2,14 @@ use std::{ffi::OsStr, io, process::Command};
 
 use crate::{CommandExt, IntoResult};
 
+pub fn command<T: AsRef<OsStr>>(path: T) -> Command {
+    let mut cmd = Command::new("/bin/open");
+    cmd.arg(path.as_ref());
+    cmd
+}
+
 pub fn that<T: AsRef<OsStr>>(path: T) -> io::Result<()> {
-    Command::new("/bin/open")
-        .arg(path.as_ref())
-        .status_without_output()
-        .into_result()
+    command(path).status_without_output().into_result()
```

Comment:
I think this `CommandExt` method should be split into two, so the configuration of IO channels is separate from executing it.

Thus, future invocations would look more like this:

```Rust
Command::new(…).without_io().status().into_result()
```